python/setup.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+from setuptools import setup
+
+setup(
+    name="translit",
```

Comment:
Is this the name of the package under which it will be published? We should spend some time bikeshedding the name. Filed a bug to discuss it: https://github.com/belaudiobooks/lacinka/issues/4 No need to do anything here.